src/pyodide/helpers.bzl

```diff
@@ -140,16 +141,25 @@ def internal_modules():
 def modules():
     return ["python-entrypoint-helper.ts"]
 
-def internal_data_modules(version):
+def internal_data_modules():
     return native.glob([
         "internal/*.py",
         "internal/patches/*.py",
         "internal/topLevelEntropy/*.py",
-    ]) + [
-        _out_path("python_stdlib.zip", version),
-        _out_path("pyodide.asm.wasm", version),
-        _out_path("emscriptenSetup.js", version),
-    ]
+    ])
+
+def pyodide_static():
+    wd_ts_bundle(
+        name = "pyodide_static",
+        eslintrc_json = "eslint.config.mjs",
+        import_name = "pyodide",
+        internal_data_modules = internal_data_modules(),
+        internal_modules = internal_modules(),
+        lint = False,
```

Comment:
True?

Reply:
Ah and then I could get rid of the extra `eslint_test` in `BUILD.bazel` since it is saying exactly what `lint=True` here would generate.

Reply:
Yeah exactly